src/ir_Bosch.cpp

```diff
@@ -117,6 +126,10 @@ uint8_t IRBosch144AC::getTemp(void) const {
   return retemp;
 }
 
+float IRBosch144AC::getTempFloat(void) const {
```

Comment:
Get rid of  `getTempFloat()` please. Put all this functionality into the `getTemp()` function, and have it return a `float` instead.